### ISSUE TITLE
ISSUE #4: Battle mode control buttons blocked by deity

### ISSUE THREAD

**Issue by JohnnyEndrizzi** [NathanCoit/4GP6Capstone#191]
During battle mode moving a capsule unit to a tile under or behind the deity makes the unit controls blocked and very difficult to click locking the match.

See attached image where the leftmost capsule is unselectable without extensive maneuvering of the camera, even through the clear sections.
https://i.imgur.com/5lUD5YH.png

A quick fix would be to remove the blank space from the button and god's hitboxes so any capsule that you can see can be clicked.  A better fix would be to have the buttons disappear when not needed.

**Comment by harrilau** [comment]
Duplicate of #68
